scripts/updates/utils/termux_pkg_is_update_needed.sh

```diff
@@ -9,20 +9,19 @@ termux_pkg_is_update_needed() {
 	local CURRENT_VERSION="$1"
 	local LATEST_VERSION="$2"
 
+	# Is this even a validly formatted version number?
+	if ! dpkg --validate-version "${LATEST_VERSION}" &> /dev/null && [[ "${CI-false}" == "true" ]]; then
+		termux_error_exit "::warning::${TERMUX_PKG_NAME:-}: $(dpkg --validate-version "${LATEST_VERSION}")"
```

Comment:
`$(dpkg --validate-version "${LATEST_VERSION}")` doesn't output anything

Reply:
> `$(dpkg --validate-version "${LATEST_VERSION}")` doesn't output anything

It does when there is an error in the version number.
<img width="1080" height="95" alt="image" src="https://github.com/user-attachments/assets/ad37398d-df9b-48b5-b43b-9a48b8e8faa1" />
But... on stderr, so I need to redirect that to stdout so it gets passed to `termux_error_exit`.